Command/MigrateCommand.php

```diff
@@ -190,136 +146,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
         
         $indexInDefaultEnv = true;
         if (strcmp($defaultEnv->getAlias(), $elasticsearchIndex) === 0 && strcmp($contentTypeNameFrom, $contentTypeNameTo) === 0) {
```

Comment:
This kind of check could be part of the interact fase.

Reply:
5e9c4b0e1954bc45fa2715255d50a6080da6c862